public/html/ticTacToe.html

```diff
@@ -0,0 +1,48 @@
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Tic Tac Toe</title>
+
+
+        <style>
```

Comment:
These styles should not be in this file. Your CSS styling should be separate from your HTML.